maven-surefire-common/src/test/java/org/apache/maven/plugin/surefire/report/NullConsoleReporterTest.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.maven.plugin.surefire.report;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.junit.Test;
+
+public class NullConsoleReporterTest {
+
+    @Test
+    public void testConstruction() throws Exception {
+        new NullConsoleReporter();
```

Comment:
The null implementations extend some existing classes which forces me to pass values to the super constructor. If the super constructor can not handle null input parameters a call the the default constructor of the null object will fail. To make sure the null objects can be created, I've added these tests. Makes sense?